scripts/keepnick.pl

```diff
@@ -1,38 +1,27 @@
-# keepnick - irssi 0.7.98.CVS 
 #
-#    $Id: keepnick.pl,v 1.19 2013/05/23 05:08:34 peder Exp $
-#
-# Copyright (C) 2001, 2002, 2006, 2013 by Peder Stray <peder@ninja.no>
+# Copyright (C) 2001-2021 by Peder Stray <peder.stray@gmail.com>
 #
 
 use strict;
 use Irssi 20011118.1727;
 use Irssi::Irc;
 
-# ======[ Script Header ]===============================================
-
 use vars qw{$VERSION %IRSSI};
 ($VERSION) = '$Revision: 1.19 $' =~ / (\d+\.\d+) /;
```

Comment:
version here too, @ailin-nemui maybe has an opinion if version needs to be updated when only cosmetic changes